examples/runner-default/main.tf

```diff
@@ -5,7 +5,7 @@ module "vpc" {
   name = "vpc-${var.environment}"
   cidr = "10.0.0.0/16"
 
-  azs             = ["eu-west-1a"]
+  azs             = ["${var.aws_region}a"]
```

Comment:
maybe even a more generic approach it the next one:

```
data "aws_availability_zones" "available" {
  state = "available"
}
module "vpc" {
  ...
  azs = ["${data.aws_availability_zones.available.names[0]}"]
}
```

What do you think?

Reply:
That looks like a much better way of doing it. I'll update my pull request with these changes.